### DISCUSSION TITLE
There's no available disk for replica , and why volume can be created?

### DISCUSSION THREAD

**Discussion by linpanru** [longhorn/longhorn#5562] [Q&A]
![image](https://user-images.githubusercontent.com/56077231/225237961-e29d2201-d2ca-4180-a46d-a301722e4d37.png)

my question is volume is always can be created successfully , even the capacity of volume is Extremely large, but when i attach this volume ,i am informed that There's no available disk, why not  tell me that the disk capacity is insufficient when i create a volume ?

**Comment by anishsedhaii** [comment]
facing similar issue.
One of the node from my RKE2 cluster had gone down, after reviving the node, the Volume/pvc that got created in the node is unable to replicate the volume across all the other nodes. 

time="2023-09-26T10:03:29Z" level=warning msg="Failed to schedule replica" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=master-212 owner=master-212 replica=pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a-r-3425efb0 state=attached volume=pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a
time="2023-09-26T10:03:29Z" level=warning msg="Failed to schedule replica" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=master-212 owner=master-212 replica=pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a-r-dc3295b5 state=attached volume=pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a
time="2023-09-26T10:03:29Z" level=warning msg="Failed to schedule replica" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=master-212 owner=master-212 replica=pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a-r-528cc79c state=attached volume=pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a
time="2023-09-26T10:03:29Z" level=error msg="There's no available disk for replica pvc-f98e4d6c-7f2f-43be-8d55-1fba2d39be5a-r-528cc79c, size 21474836480"


All the nodes have enough storage and volumes that are created in other nodes can be replicated on this node, but volume created on this node gives above error after trying to replicate.

**Comment by derekbit** [comment]
@linpanru @anishsedhaii 
Please provide a support bundle. Thanks.
